components/atoms/typography.styles.ts

```diff
@@ -52,7 +56,8 @@ export const DefaultParagraph = styled.p`
     font-weight: 400;
     font-size: 16px;
     line-height: 140%;
-    color: ${Grey010};
+    margin: 0;
+    color: ${(props: TypographyStyleProps) => props.color};
```

Comment:
you can  do 
```
yarn add -D @types/styled-components
```
and after

```
export const DefaultParagraph = styled.p<TypographyStyleProps>`
    font-family: 'Figtree', sans-serif;
    font-style: normal;
    font-weight: 400;
    font-size: 16px;
    line-height: 140%;
    margin: 0;
    color: ${({ color }) => color};
`;
```

Reply:
This way if you have multiple props, it is easier to handle them with destructuring

Reply:
coolio, thanks